### PR TITLE
store_metadata_in_osv3: log extra debug info on failure

### PR DESCRIPTION
**Pull request by twaugh**

### CODE REVIEW

File: atomic_reactor/plugins/exit_store_metadata_in_osv3.py [hunk 1]
```diff
@@ -188,10 +188,17 @@ def run(self):
...
+            self.log.debug("annotations: %r", annotations)
+            raise
 
         labels = self.make_labels()
         if labels:
-            osbs.update_labels_on_build(build_id, labels)
+            try:
+                osbs.update_labels_on_build(build_id, labels)
+            except OsbsResponseException:
+                self.log.debug("labels: %r", labels)
```
Comment (lcarva) [inline atomic_reactor/plugins/exit_store_metadata_in_osv3.py]: raise?

Comment (twaugh) [inline atomic_reactor/plugins/exit_store_metadata_in_osv3.py]: Indeed. Fixed.

**Comment by vrutkovs** [comment]
LGTM

**Comment by mmilata** [comment]
LGTM, merged.

**Pull request closed without merging**